tools/scripts/darwin.sh

```diff
@@ -58,7 +58,7 @@ sed "s/${OLDSHA256CHECKSUM}/${SHA256CHECKSUM}/" ../tools/files/darwin/logdna-age
 cd ..
 
 # STEP 4: RELEASE
-ghr -draft \
```

Comment:
just curious, why is the `draft` step being removed? does this make the `pause` below unnecessary, and/or should we keep the current approval process?

Reply:
`-draft` creates a new `GitHub Draft Release` - I am gonna create a `GItHub Release` manually and since running these scripts manually, I wanna test whether this just attaches the artifacts to already existing release or not

Reply:
a draft is the same as a "normal" release, just without it being officially published. it's good for testing things like you mentioned safely without publishing a new release, and it can be published later if testing goes well.

that said, no problem from me if you're ok with the process.

Reply:
@mdeltito, yea, we will visit that later in the case of automation - right now this is pretty experimental and manual with just some set of automation